src/seth/libexec/seth/seth-selector

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
duplicate of `seth sig`?

Reply:
Heh, yes, didn't see that one. Will remove this one

Reply:
one thing that always bothered me about `sig`, it's very literal... `withdraw(uint)` is not the same as `withdraw(uint256)`, whereas other functions will correctly account for this, such as `seth call 0x.. 'something(uint)' 0x0`. But can be left for another day :P

Reply:
> one thing that always bothered me about `sig`, it's very literal... `withdraw(uint)` is not the same as `withdraw(uint256)`, whereas other functions will correctly account for this, such as `seth call 0x.. 'something(uint)' 0x0`. But can be left for another day :P

would it be possible to fix this just by replacing uses of standalone `uint` with `uint256`? It's the only type with that behavior iirc

Reply:
I'll look into how `seth call` does it, but worst case it should be easy to find/replace `uint` and `int` with `uint256` and `int256` before hashing

Reply:
the `memory` bs also screws it up

Reply:
Good call, just pushed a version of `seth sig` that will behave the same as `seth call`